addons/point_of_sale/static/src/xml/Screens/ProductScreen/Orderline.xml

```diff
@@ -59,10 +59,10 @@
                         discount
                     </li>
                 </t>
-                <t t-if="customerNote">
+                <t t-foreach="customerNote" t-as="note">
```

Comment:
Can you rename this such that it reflects the fact that it is a list? Maybe `customerNotes`.

Also, given that we use `customerNote` as a string everywhere in the pos, it might be simpler to just concatenate all the notes in one string ( maybe with some separator, ex: `--` )
Otherwise this will require a lot of changes in the forward ports
What do you think?

Reply:
Yes that makes sense, I wil modify it.